sdk/webpubsub/Microsoft.Azure.WebJobs.Extensions.WebPubSub/README.md

```diff
@@ -4,7 +4,7 @@ This extension provides functionality for receiving Web PubSub webhook calls in
 
 [Source code](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/webpubsub/Microsoft.Azure.WebJobs.Extensions.WebPubSub/src) |
 [Package](https://www.nuget.org/packages/Microsoft.Azure.WebJobs.Extensions.WebPubSub) |
-[API reference documentation](https://azure.github.io/azure-webpubsub/references/functions-bindings) |
+[API reference documentation](https://docs.microsoft.com/dotnet/api/microsoft.azure.webjobs.extensions.webpubsub) |
```

Comment:
The previous link was broken.  _([ref](https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1679159&view=logs&j=d41e9505-d81b-5b3c-5b50-be71bc9b0ccb&t=2521f384-3a28-5bc9-41e5-2f3e6ef99ecd))_ 

I adjusted this to follow the pattern used by other libraries, pointing the API reference to the MS Docs content.